### ISSUE TITLE
Throw error when pagination with `after`

### ISSUE THREAD

**Issue by trungtin** [graphile/graphile-build#97]
https://github.com/graphile/graphile-build/blob/db30a95d80b5eca17b78cdedda63b778b9463c37/packages/graphile-build-pg/src/queryFromResolveData.js#L135-L137

With this simple query:
```
query getMyTasks($taskCondition: TaskCondition!) {
  allTasks(condition:$taskCondition, after:"WyJ0YXNrcyIsIndkTHdiIl0=") {
    totalCount
    nodes {
      name
      nodeId
      id
    }
  }
}
```
It will throw Error because `cursorValue[getPgCursorPrefix().length]` is (id) not [(id)].
Example `cursorValue` at this point is:
```
['tasks', 'wdLwb']
```
Simply change to:
```
const cursor = cursorValue[getPgCursorPrefix().length]
const sqlCursors = (Array.isArray(cursor) ? cursor : [cursor]).map(val =>
  sql.value(val)
);
```
work though. Have no idea whether I have set thing up wrongly or anything else.

**Comment by benjie** [comment]
Ah yeah, this is likely due to a compatibility hack I had to do to make the cursors line up between v4 and v3. Cursors come from:

https://github.com/graphile/graphile-build/blob/731d2239c1da47a60765ba64cd55cefe71f085fe/packages/graphile-build-pg/src/queryFromResolveData.js#L110-L128

It's line 126 above that's introducing the issue I think because the value is not a (nested) array - this is the compat hack I had to do - without it I broke a lot of acceptance tests.

However, I would not expect this to hit that line because there should be an implicit orderBy in the query you're running. If you add an explicit orderBy does the problem go away? I'm wondering if you've not defined a primary key on the tasks table? This is still a bug if so, but it helps explain why it's occurring and why it wasn't caught earlier. If this isn't the case then I'm somewhat confused as to what has caused it.

I'd encourage a PR; in it I'd want to see:

- [ ] the fix above
- [ ] an acceptance test that would fail with the current codebase but passes with the fix (just add a query file to this folder that reproduces the issue: https://github.com/graphile/graphile-build/tree/master/packages/postgraphile-core/__tests__/fixtures/queries - if you cannot then you might need to add another table to [the schema file](https://github.com/graphile/graphile-build/blob/master/packages/postgraphile-core/__tests__/kitchen-sink-schema.sql))
- [ ] tests passing on CI
